tests/format/angular/angular/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1149,9 +1151,11 @@ printWidth: 1
       $event)
   "
   *ngIf="
-    something
-      ? true
-      : false
+    (
+      something
+    ) ?
+      true
+    : false
```

Comment:
I thought this PR is only add indentation back?

I don't like this.

Reply:
That was the previous PR, which is nicer in some cases but leaves a lot of problems unsolved. You can read through some of the discussion to see why.

Yes, like I said, this will rub some people the wrong way at first 😄 I recommend you try it on a codebase you work on regularly for a few days before leaping to a conclusion… is that something you could try?

Reply:
This will almost certainly make things worse. There's obvious disagreement about how to do ternaries but people seemed much more okay with the old way of doing ternaries than the new way. Adding yet another third that has not received any sort of public support seems very risky. The options that have received the most support are [here](https://github.com/prettier/prettier/issues/5814#issuecomment-468978755) and [here](https://github.com/prettier/prettier/issues/9561#issuecomment-751684197) which are both very similar to the old way of doing it. Can we just go with one of those?

Reply:
This PR definitely does need a thorough justification for why I think this nonstandard, unfamiliar thing is best. I've been wanting to write up that justification for a year or so and haven't found the time. I should be able to soon, and I'd really appreciate help from the other prettier maintainers (or someone in the community who wants to see ternaries get better and has an open mind) in making it happen.

Reply:
The parenthesis are there because the test has a print width of 1 which isn't what people would experience. We should update the test to use a normal print width but this will be for another commit.